### DISCUSSION TITLE
The index() reset the number for every page

### DISCUSSION THREAD

**Discussion by Unisolgr14** [Power-Components/livewire-powergrid#1759] [Q&A]
I am using 4.x and index() function as in the documentation to have a serial number .

`Column::make('Dish name', 'name')
    ->index(), `
The issue is it regenerate stating from 1 for each page (using pagination)

**Comment by zikrimuhammad09** [comment]
up

**Comment by luanfreitasdev** [comment]
Yes, this is the default behavior of an index because $loop->index is used under the hood.

**Reply by Unisolgr14** [reply]
How to stop the resetting of the index number , I mean to stop this default behavior and have continuous serial number without resetting for each page
